sphinx/builders/latex/constants.py

```diff
@@ -11,7 +11,12 @@
 from typing import Any, Dict
 
 PDFLATEX_DEFAULT_FONTPKG = r'''
-\usepackage{times}
+\usepackage{tgtermes}
+\usepackage{tgheros}
+\renewcommand{\ttdefault}{txtt}
```

Comment:
as commented above, tgtermes and tgheros render almost 100% like what package times gave for the body font and the sans font. I hesitated about using tgcursor for the monospace font, but really Courier is very bad match to Times-like text font. So bad we had to use ``\small`` LaTeX command in verbatim because it is too wide. The ``txtt`` font is a very nice monospace font, only a bit wider than times-like clones, and even at normal size it it is still a bit narrower than the Courier at small size. So we gain a bit in code-blocks and the other advantage is that if a code-block is an section rendered via a smaller size, either from raw latex mark-up or from being in a footnote, then the size will match the one of the text, due to ``\fvset{fontsize=auto}``